force-app/main/default/classes/SOQL.cls

```diff
@@ -201,7 +248,13 @@ public with sharing class SOQL {
      * @return A list of Cases
      */
     public static List<Case> getCasesWithStatus() {
-        return null; // Replace null with the variable you used to store the result
+        List<Case> caseList = new List<Case>([SELECT Id, Status, CreatedDate 
+                                                FROM Case 
+                                                WHERE Status != 'Closed' 
```

Comment:
Although the test method uses `Open` -- it's good that you used `!= 'Closed'` since this allows for more Status picklist values.